cypress/integration/work-order/variation-authorisation.spec.js

```diff
@@ -72,4 +96,87 @@ describe('Contract manager can authorise variation', () => {
 
     cy.contains('You have approved a variation for work order 10000012')
   })
+
+  // summary page and calculation
+  it('shows summary page and calculation of variation cost', () => {
+    cy.visit(`${Cypress.env('HOST')}/work-orders/10000012`)
+    cy.contains('Authorise Works Order Variation')
+      .should(
+        'have.attr',
+        'href',
+        '/work-orders/10000012/variation-authorisation'
+      )
+      .click()
+
+    cy.contains('Authorisation variation request: 10000012')
+    cy.contains('Summary of Tasks and SORs')
+
+    cy.get('.origina-sor-summary').within(() => {
+      cy.contains('td', 'DES5R006 - Urgent call outs')
+      cy.contains('td', '1')
+      cy.contains('td', '£10')
+    })
+
+    cy.contains('Updated Tasks SORs')
+    cy.contains('Updated by: John Johnson (Alphatrack)')
+    cy.contains('Tuesday, 11 May 2021')
+
+    cy.contains('Variation reason: More work needed')
+
+    cy.get('.updated-tasks-table').within(() => {
+      // Increased task
+      cy.contains('td', 'Increase')
+      cy.contains('td', 'DES5R005')
+      cy.contains('p', 'Normal Call outs')
+      cy.contains('td', '£4')
+      cy.contains('td', '1')
+      cy.contains('td', '£4')
+      cy.contains('td', '4000')
+      cy.contains('td', '£1600')
+
+      //Reduced task
+      cy.contains('td', 'Reduced')
+      cy.contains('td', 'DES5R006')
+      cy.contains('p', 'Normal Call outs')
+      cy.contains('td', '£19')
+      cy.contains('td', '10')
+      cy.contains('td', '£190')
+      cy.contains('td', '2')
+      cy.contains('td', '£38')
+
+      //New task
+      cy.contains('td', 'Reduced')
+      cy.contains('td', 'DES5R007')
+      cy.contains('p', 'Normal Call outs')
+      cy.contains('td', '£25')
+      cy.contains('td', '0')
+      cy.contains('td', '£0')
+      cy.contains('td', '2')
+      cy.contains('td', '£50')
+
+      //Unchanged task
+      cy.contains('td', 'Unchanged')
+      cy.contains('td', 'DES5R006')
+      cy.contains('p', 'Normal Call outs')
+      cy.contains('td', '£10')
+      cy.contains('td', '1')
+      cy.contains('td', '£10')
+      cy.contains('td', '1')
+      cy.contains('td', '£10')
```

Comment:
Could we possible scope this with data-attributes?
i.e. we should so something like `cy.get('[data-row-id="2"]').within(() => {..})` otherwise these matching values are too generic for the entire page